### ISSUE TITLE
Testing only - Verify connectivity in Azure

### ISSUE THREAD

**Issue by menib** [QualiSystems/Azure-Shell#35]

**Comment by yairbb** [comment]
Reminder to check the connectivity between the machines after deployment from Azure side.

**Comment by yairbb** [comment]
Duplicate of #179

**Comment by tornizanQ** [comment]
[test plan](http://team:8080/tfs/Quali-Collection/QualiSystems/_testmanagement#planId=125053&suiteId=158027)

**Comment by tornizanQ** [comment]
blocked by bug 
https://github.com/QualiSystems/Azure-Shell/issues/276
